spec/features/update_users_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature "Update User", type: :feature do
+  
+  scenario 'successfully' do
+    user = create(:user)
+    
+    visit '/users/sign_in'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: "password"
+    click_button 'Log in'
+    
+    click_link 'Edit profile'
+    
+    expect(page).to have_css('h2', text: 'Edit Profile')
+    fill_in 'First name', with: "New-name"
+    click_button 'Update profile'
+    
```

Comment:
@ianroberts131 If this is the test you mentioned, you might try debugging it by printing the page html and page text and seeing if there are any clues in the output as to why the profile is not getting updated:

```
# Try adding these lines after line 17, re-run the spec, and
# read the printed page content in the terminal:
puts "The page text is"
puts page.text
puts "The page HTML is"
puts page.html
```